src/super_gradients/training/models/detection_models/csp_darknet53.py

```diff
@@ -77,14 +77,29 @@ def fuseforward(self, x):
         return self.act(self.conv(x))
 
 
+class DepthWiseConv(nn.Module):
+    # STANDARD CONVOLUTION
+    def __init__(self, input_channels, output_channels, kernel, stride, activation_type: Type[nn.Module],
+                 padding: int = None):
+        super().__init__()
+
+        self.dconv = Conv(input_channels, input_channels, kernel, stride, activation_type, padding, input_channels)
```

Comment:
can you please add the arg name `groups=imput_channels`

Reply:
Sure, added in d17de50